<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="MagnitudeSpinBox" name="magResonanceFreq">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="MagnitudeSpinBox" name="magICoeff">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output Frequency</string>
     </property>
     <property name="buddy">
      <cstring>magOutputFreq</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Output Amplitude</string>
     </property>
     <property name="buddy">
      <cstring>magOutputAmpl</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="MagnitudeSpinBox" name="magOutputFreq">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="MagnitudeSpinBox" name="magOutputAmpl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Prop Coefficient</string>
     </property>
     <property name="buddy">
      <cstring>magPCoeff</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="refGTcombLineBox">
       <property name="text">
        <string>Ref &gt; Comb Line</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="autoLockButton">
       <property name="toolTip">
        <string>Auto Lock</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="RepetitionRate.qrc">
         <normaloff>:/openicon/icons/lock-pink.png</normaloff>:/openicon/icons/lock-pink.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lockButton">
       <property name="toolTip">
        <string>Manual Lock</string>
       </property>
       <property name="text">
        <string>Lock</string>
       </property>
       <property name="icon">
        <iconset resource="RepetitionRate.qrc">
         <normaloff>:/openicon/icons/lock-4.png</normaloff>:/openicon/icons/lock-4.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="unlockButton">
       <property name="toolTip">
        <string>Unlock</string>
       </property>
       <property name="text">
        <string>Unlock</string>
       </property>
       <property name="icon">
        <iconset resource="RepetitionRate.qrc">
         <normaloff>:/openicon/icons/media-playback-stop-7.png</normaloff>:/openicon/icons/media-playback-stop-7.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="1">
    <widget class="QComboBox" name="harmonicOutputCombo"/>
   </item>
   <item row="13" column="1">
    <widget class="QComboBox" name="filterCombo"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Offset Frequency</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="MagnitudeSpinBox" name="magReferenceAmpl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="MagnitudeSpinBox" name="magOffsetFreq">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Reference Frequency</string>
     </property>
     <property name="buddy">
      <cstring>magReferenceFreq</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Reference Amplitude</string>
     </property>
     <property name="buddy">
      <cstring>magReferenceAmpl</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="MagnitudeSpinBox" name="magOffset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="MagnitudeSpinBox" name="magHarmonic">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Integrator Coefficient</string>
     </property>
     <property name="buddy">
      <cstring>magICoeff</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="MagnitudeSpinBox" name="magPCoeff">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MagnitudeSpinBox" name="magReferenceFreq">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Offset</string>
     </property>
     <property name="buddy">
      <cstring>magOffset</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Harmonic Output</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Resonance Frequency</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="MagnitudeSpinBox" name="magDCThreshold">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QCheckBox" name="dcThresholdBox">
     <property name="text">
      <string>DC Threshold</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Integer Harmonic</string>
     </property>
     <property name="buddy">
      <cstring>magOutputAmpl</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Errorsig Harmonic</string>
     </property>
     <property name="buddy">
      <cstring>magOutputAmpl</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="MagnitudeSpinBox" name="magErrorsigHarmonic">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Errorsignal generated from this harmonic</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Harmonic Reference</string>
     </property>
     <property name="buddy">
      <cstring>magOutputAmpl</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="MagnitudeSpinBox" name="magHarmonicReference">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>frequency shift from additional mixing step</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnitudeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>uiModules.MagnitudeSpinBox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>magReferenceFreq</tabstop>
  <tabstop>magReferenceAmpl</tabstop>
  <tabstop>magOutputFreq</tabstop>
  <tabstop>magOutputAmpl</tabstop>
  <tabstop>magHarmonic</tabstop>
  <tabstop>magOffset</tabstop>
  <tabstop>magPCoeff</tabstop>
  <tabstop>magICoeff</tabstop>
  <tabstop>lockButton</tabstop>
  <tabstop>unlockButton</tabstop>
 </tabstops>
 <resources>
  <include location="RepetitionRate.qrc"/>
 </resources>
 <connections/>
</ui>
